app.css

```diff
@@ -237,3 +237,9 @@ input:checked + .slider:before {
   transform: translateX(1rem);
 }
 /* END SLIDER STYLES*/
+/* BEGIN FILTER STYLES*/
+
+aside .filter {
+  height: calc(100vh - 338px - 4rem);
```

Comment:
toto asi nebude fungovat vzdy nie ?

Ja by som zvazil ci nemat scrollovatelny cely ten bok aj s kalendarom, nie len filtre.

Reply:
to, co pise zajo, ze nech sa to scrolluje cele je podla mna lepsi napad

Reply:
a na ľavo alebo na pravo? (verím, že to bude fungovať naozaj vždy, ten kalendár widget má všetky veci v pixeloch a tomu topbaru som zafixoval veľkosť na 4rem (predtým to bolo závyslé od výšky písma, toto by malo stále fungovať, aj keď zväčšíš písmo) )

Reply:
akože, scrollbar je štandardne v pravo, tam by som ho ako user očakával asi :thinking:

Reply:
Netvrdím, že nie ale dal som to ako otázku na diskusiu